packages/tools/devtools/devtools-view/src/components/data-visualization/ToolTipContentsView.tsx

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { type VisualChildNode, VisualNodeKind } from "@fluidframework/devtools-core/internal";
+import React from "react";
+
+/**
+ * TODO
+ */
+export interface ToolTipContentsViewProps {
+	contents: Record<string, VisualChildNode> | string;
+}
+
+/**
+ * TODO
+ */
+export function ToolTipContentsView(props: ToolTipContentsViewProps): React.ReactElement {
+	const { contents } = props;
+
+	if (contents === undefined) {
+		return <></>;
+	}
+
+	if (typeof contents === "string") {
+		return <div> {contents} </div>;
+	}
+
+	const listItems: React.ReactElement[] = [];
+	let listItem: React.ReactElement;
+
+	for (const contentsValue of Object.values(contents)) {
+		if (contentsValue.nodeKind === VisualNodeKind.TreeNode) {
+			for (const [fieldKey, fieldValue] of Object.entries(contentsValue.children)) {
+				listItem =
+					fieldValue.nodeKind === VisualNodeKind.ValueNode ? (
+						<li key={fieldKey}>
+							{fieldKey} : {fieldValue.value}
+						</li>
+					) : (
+						<li>Unsupported Data Structure!</li>
```

Comment:
This isn't correct. The input tree is allowed to be recursive. If we encounter a recursive tree, we probably need to call this component recursively to created nested sublists.

Reply:
I suppose the idea is for this to be a short term solution. In that case, let's add some documentation noting that this is a known issue here, but it's only meant to be temporary and we should replace this entire component with a tree view later.

Reply:
I would also recommend putting this text in italics or something to highlight that it doesn't represent actual data. And I would omit the `!` at the end 🙂